partials/map.php

```diff
@@ -1,7 +1,13 @@
 <div id="map">
-  <?php
-    get_template_part('partials/background-shape');
-  ?>
+  <div id="background-pattern-holder">
+    <?php
+      echo url_get_contents(get_template_directory_uri() . '/dist/static/patterns/pattern-1.svg');
```

Comment:
not suggesting changing this code but rather bringing in these improvements to `url_get_contents` https://github.com/interglobalvision/globie-wordpress-starter-pack/pull/112